client/my-sites/backup/rewind-flow/restore.tsx

```diff
@@ -80,6 +80,8 @@ const BackupRestoreFlow: FunctionComponent< Props > = ( {
 
 	const [ userHasRequestedRestore, setUserHasRequestedRestore ] = useState< boolean >( false );
 	const [ restoreInitiated, setRestoreInitiated ] = useState( false );
+	const [ restoreFailed, setRestoreFailed ] = useState( false );
```

Comment:
This restore flow is a good candidate for refactoring. I think we could benefit from handling the different restore states as a state machine in Redux. I started drafting some of it but preferred to focus on fixing the immediate bug. I’m leaving this note to remind myself to revisit this.